### ISSUE TITLE
&#64; bug

### ISSUE THREAD

**Issue by sprynm** [straker/livingcss#57]
When I use something like 
```
* @code {sass}
* &#64;include myMixin(); 
* &#64;include myMixin(option);  
```
I'm getting this 
```
@include myMixin(); 
&#64;include myMixin(option);  
```

Only the first `&#64;` is getting replaced with the @ symbol.

Any ideas?

**Comment by straker** [comment]
It would seem I forgot to use a global replace for that in the code https://github.com/straker/livingcss/blob/master/lib/tags.js#L261.

If that was changed to a simple regex with a global flag it should fix the problem

```js
this.block.code.description = this.block.code.description.replace(/&#64;/g, '@');
```

**Comment by straker** [comment]
fixed in v4.5.1
